core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1474,7 +1469,7 @@ FROM (
 	FOR UPDATE OF e1
 ) e0
 WHERE e0.id = evm.txes.id
-RETURNING e0.id, e0.nonce`, ErrCouldNotGetReceipt, cutoff, chainID.String())
+RETURNING e0.id, e0.nonce`, ErrCouldNotGetReceipt, latestFinalizedBlockNum, chainID.String())
```

Comment:
Do we want `HAVING max(evm.tx_attempts.broadcast_before_block_num) < $2` to be inclusive?